_includes/configuration/content-langs.md

```diff
@@ -0,0 +1,87 @@
+{% if page.name == "tinymcespellchecker.md" %}
+### `content_langs`
+{% else %}
+## `content_langs`
+{% endif %}
+
+{{ site.requires_5_9v }}
+
+This option sets the languages displayed in the `language` dropdown toolbar button and the `language` menu item. The option accepts an array of language objects. Each language object is a JavaScript object defined using the following properties.
+
+| Name | Type | Requirement | Description |
+| ---- | ---- | ----------- | ----------- |
+| `title` | String | required | The name shown for this language in the UI. |
+| `code` | String | required | The language tag, as defined in [Tags for Identifying Languages (BCP47)](https://www.ietf.org/rfc/bcp/bcp47.txt), for this language. This tag is placed in the `lang` attribute for elements formatted using this language. |
+| `customCode` | String | optional | An alternative language tag, that does not need to be compatible with BCP47. This code is placed in the `data-mce-lang` attribute for elements formatted using this language, if it is defined. |
+
+**Type:** `Array`
+
+{% if page.name == "tinymcespellchecker.md" %}
+**Default:**
+
+```js
+[
+    { title: 'English', code: 'en' },
+    { title: 'Spanish', code: 'es' },
+    { title: 'French', code: 'fr' },
+    { title: 'German', code: 'de' },
+    { title: 'Portuguese', code: 'pt' },
+    { title: 'Chinese', code: 'zh' }
+]
+```
+
+> **Note:** This default is provided by the Spell Checker Pro plugin, it is not available in the core editor.
+
+> **Note:** The languages specified by the deprecated [`spellchecker_select_language`](#spellchecker_select_language) option are used as a fallback if they are available.
+
+{% else %}
+There is no default value for the `content_langs` option. If no value is specified, the `language` toolbar button and menu item are not available. A default value for this option is provided by the Spell Checker Pro plugin. For information on using the `content_langs` option with the Spell Checker Pro plugin, see: [Spell Checker Pro plugin - `content_langs`]({{site.baseurl}}/plugins/premium/tinymcespellchecker/#content_langs).
```

Comment:
This whole section reads really oddly when read in the rendered view, as it just feels as though it's coming out of nowhere... This really should be before we specify the `type` as nowhere else in the docs do we include additional information after the `type`.

I wonder if the structure should be something like this instead, as that's similar to what we do in the UI components pages:

- Intro para
- this para
- type
- Spec heading and table
- example

Reply:
I've moved this paragraph earlier, but I don't think putting it before the type / spec table is going to work without some heavier reworking. Let me know if you still find this flow hard to follow, and I'll shuffle things again